pkgs/by-name/re/renpy/package.nix

```diff
@@ -34,16 +34,16 @@ stdenv.mkDerivation {
     owner = "renpy";
     repo = "renpy";
     rev = version;
-    hash = "sha256-07Hj8mJGR0+Pn1DQ+sK5YQ3x3CTMsZ5h5yEoz44b2TM=";
+    hash = "sha256-k8mcDzaFngRF3Xl9cinUFU0T9sjxNIVrECUguARJVZ4=";
   };
 
   nativeBuildInputs = [
     pkg-config
     makeWrapper
     # Ren'Py currently does not compile on Cython 3.x.
     # See https://github.com/renpy/renpy/issues/5359
-    python3.pkgs.cython_0
-    python3.pkgs.setuptools
+    python311.pkgs.cython_0
```

Comment:
Is newer Cython still not supported?

Reply:
Seems like cython 2 should work. See https://github.com/renpy/renpy/issues/5359#issuecomment-1943005306 
But `python311Packages.cython` has version  `3.0.10`.